datafusion/sqllogictest/test_files/array.slt

```diff
@@ -2427,6 +2469,18 @@ NULL 10
 NULL 10
 NULL 10
 
+query II
+select array_length(arrow_cast(array[array[1, 2], array[3, 4]], 'LargeList(List(Int64))'), column3), array_length(arrow_cast(column1, 'LargeList(Int64)'), 1) from arrays_values;
```

Comment:
mixing these non-table with table test cases is quite confusing, maybe separate them?